### PR TITLE
Enable system reconcilation

### PR DESCRIPTION
**Pull request by MarcinGinszt**
This pull request allows reconciliation of systems- issue https://github.com/OpsLevel/kubectl-opslevel/issues/171
It is built on top of https://github.com/OpsLevel/kubectl-opslevel/pull/172

### CODE REVIEW

**Comment by rocktavious** [comment]
@MarcinGinszt - does this need to be rebased on the main branch?  It seems to still be showing changes from your other PR

**Comment by MarcinGinszt** [comment]
@rocktavious my bad, sorry! Rebased it. Please, let me know if there is anything else I could do here.

**Comment by rocktavious** [comment]
@MarcinGinszt - I wanted to let you know we have finally been able to priortize this repo and i'm going to be doing a pretty big refactor to increase the test coverage as well as make the repo more simple.  Unfortuntely because of that refactor this code might not be the same and it also might mean massive conflict resolution to include this PR.  So what i'm going to do is keep this PR open for reference but after the refactor i'll make sure this code is included.

Thanks for doing the legwork on this one and sorry we havn't been responsive in being able to get this merged and shipped.  We expect to have the refactor shipped in 3-4 weeks.

File: src/common/client.go [hunk 1]
```diff
@@ -318,3 +319,21 @@ func handleRepositories(client *opslevel.Client, registration ServiceRegistratio
...
+
+func handleSystem(client *opslevel.Client, registration ServiceRegistration, service *opslevel.Service) {
+	if registration.System != "" {
+		system, foundSystemErr := client.GetSystem(registration.System)
+		if foundSystemErr != nil {
+			log.Warn().Msgf("[%s] System with alias: '%s' not found so service cannot be attached to it ... skipping", service.Name, system)
+			return
+		}
+
+		assignServiceErr := system.AssignService(client, service.Aliases[0])
```
Comment (matthewhughes-uw) [inline src/common/client.go:331]: I thnk using the Id to uniquely identify might be clearer.g. https://github.com/OpsLevel/kubectl-opslevel/blob/692346d87380b028c8197834e2bc49ad36faef88/src/common/client.go#L239

```suggestion
		assignServiceErr := system.AssignService(client, string(service.Id))
```

**Pull request closed without merging**